app/knight_items/knight_manipulation.py

```diff
@@ -0,0 +1,168 @@
+import random
+from typing import List, Dict, Tuple, Any
+
+from app.event_items.event_master import Event
+from app.knight_items import libraries
+
+knights_obj_list = []
+
+
+class Knight:
+    def __init__(self,
+                 name: str,
+                 power: int,
+                 hp: int,
+                 armour: List[Dict[str, Any]],
+                 weapon: Dict[str, Any],
+                 potion: Dict[str, Any]) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.armour = armour
+        self.weapon = weapon
+        self.potion = potion
+
+
+def knight_dict_creation() -> Dict[str, Dict[str, Any]]:
+    knights_full_equip = {}
+    for name in libraries.names_of_knights:
+        if name == "arthur":
+            name = "Artur"
+
+        knights_full_equip[name] = {
+            "name": name.replace("_", " ").title(),
+            "power": random.randint(50, 55),
+            "hp": random.randint(200, 220),
```

Comment:
Since the assignment says "P.S: You can add your logic to this project if you want: just be creative here" I decided to add whatever I wanted, provided that the tests passed